src/qtiCommonRenderer/helpers/uploadMime.js

```diff
@@ -45,54 +47,45 @@ var uploadMime = {
             return;
         }
 
-        if (types.length === 1) {
-            //if there is only one value set into the qti standard type attribute
-            if (types[0] !== 'any/kind') {
-                interaction.attr('type', types[0]);
-            }
-        } else {
-            //if there is more than one value, set into into TAO specific css classes
-            //qti 2.1 xsd indeed does not allow comma-separated multi mime type value for the attribute "type
+        //For backward compatibility:
+        //if there is more than one value, set into into TAO specific css classes
+        if (types.length > 1) {
             interaction.attr(
                 'class',
                 _.reduce(
                     types,
-                    function(acc, selectedType) {
+                    function (acc, selectedType) {
                         return acc + ' x-tao-upload-type-' + selectedType.replace('/', '_');
                     },
                     classes
                 ).trim()
             );
         }
+
+        interaction.attr('type', types.join(delimetr));
```

Comment:
For case `types.length === 1` no call `interaction.attr('type', types[0]);`

Reply:
For any length `interaction.attr('type', types.join(delimetr));` will be called which equals to `interaction.attr('type', types[0]);`